CDP4PluginPackager/Models/AutoGen/Target.cs

```diff
@@ -0,0 +1,26 @@
+﻿// -------------------------------------------------------------------------------------------------
+// <copyright file="Target.cs" company="RHEA System S.A.">
+//    Copyright (c) 2015-2020 RHEA System S.A.
+// </copyright>
+// <summary>
+//   This is an auto-generated class. Any manual changes on this file will be overwritten!
+// </summary>
+// -------------------------------------------------------------------------------------------------
+
```

Comment:
This file has been generated using a csproj file with this tool to convert Xml structure into Csharp : <link>